cicecore/cicedynB/general/ice_init.F90

```diff
@@ -1017,6 +1068,36 @@ subroutine input_data
          endif
       endif
 
+      ! Implicit solver input validation
```

Comment:
shouldn't these error checks and aborts be wrapped in kdyn=3?

Reply:
yes, I guess that doesn't hurt. Good idea, thanks.

Reply:
I've pushed a new commit that adds this.